rs/nervous_system/integration_tests/tests/advance_target_version.rs

```diff
@@ -0,0 +1,141 @@
+use ic_nervous_system_integration_tests::pocket_ic_helpers::sns;
+use ic_nervous_system_integration_tests::{
+    create_service_nervous_system_builder::CreateServiceNervousSystemBuilder,
+    pocket_ic_helpers::{
+        add_wasm_via_nns_proposal, add_wasms_to_sns_wasm, install_nns_canisters, nns,
+    },
+};
+use ic_nns_test_utils::sns_wasm::create_modified_sns_wasm;
+use ic_sns_governance::pb::v1 as sns_pb;
+use ic_sns_swap::pb::v1::Lifecycle;
+use ic_sns_wasm::pb::v1::SnsCanisterType;
+use pocket_ic::PocketIcBuilder;
+use std::time::Duration;
+
+const TICKS_PER_TASK: u64 = 2;
+
+#[test]
+fn test_get_upgrade_journal() {
```

Comment:
This test doesn't really seem like it belongs in nervous_system - it seems more specific to SNS governance, no?

Reply:
It tests an interaction between SNS Gov and SNS-W, so I think it seems reasonable to have in nervous_system. Additionally the helpers in nervous_system are useful for this test I think, but maybe equivalent helpers exist in the SNS Governance-specific integration tests

Reply:
https://github.com/dfinity/ic/pull/1859 ← I implemented it in this PR, but I think down the line we should make a directory in sns-integration-tests for governance-related tests. Happy to do that too, but it would add a lot to this PR, which is why I made a separate one